analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -596,6 +596,15 @@ class Pip(
                 }
             }
 
+            val projectUrls = pkgInfo["project_urls"]
+            val vcsFallbackUrls = listOfNotNull(
+                homepageUrl,
+                pkgInfo["project_url"]?.textValue(),
+                projectUrls["Code"]?.textValue(),
+                projectUrls["Homepage"]?.textValue(),
+                projectUrls["Source"]?.textValue()
```

Comment:
@sschuberth In #509 you also mention "Source Code" as a possible field, I haven't seen this being used anywhere, shall I add it anyway?

Reply:
Yes please. A quick search shows that "Source Code" is e.g. used at https://github.com/lepture/authlib/blob/c73e2a8921a15ee1f96b604123e49bd2d5fd651b/setup.cfg#L38 (via `setup.cfg`).